apps/explorer/src/app/components/markets/markets-table.tsx

```diff
@@ -138,3 +137,20 @@ export const MarketsTable = ({ data }: MarketsTableProps) => {
     />
   );
 };
+
+const getAsset = (
+  data: MarketFieldsFragment
+): {
+  decimals: number | undefined;
+  symbol: string;
+  id: string;
+} => {
+  // TODO to handle baseAsset for Spots
+  return 'settlementAsset' in data.tradableInstrument.instrument.product
```

Comment:
I'd use a condition using product.__typename. Also, it looks like this function would be best placed as a utility function, probably libs/markets?